TUTORIAL.md

```diff
@@ -2,62 +2,73 @@
 
 `cargo install voidmap`
 
-if you don't have cargo, an easy way to get and manage
-it is [via rustup](https://www.rustup.rs/). Ensure that `~/.cargo/bin`
-is in your `$PATH` afterward, so that you can use the `rustup` and `cargo`
+if you don't have cargo, an easy way to get and manage it is [via
+rustup](https://www.rustup.rs/). Ensure that `~/.cargo/bin` is in your
+`$PATH` afterward, so that you can use the `rustup` and `cargo`
```

Comment:
if you are going to add linebreaks, i suggest breaking at the end of a sentence.
that makes future editing easier on the diffs

Reply:
I used pandoc as an autoformatter, and it sets hard linebreaks by default. Soft may well be the better idea here.